dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/event/data/EventAnalyticsServiceTest.java

```diff
@@ -225,17 +226,19 @@ public void tearDownTest()
         analyticsTableGenerator.dropTables();
     }
 
+    @Ignore // FIXME jimgrace This test returns no grid rows, so it fails now.
```

Comment:
(See second paragraph of "Testing", above.) I have now returned the EventAnalyticsServiceTest to the way it was, which reports success but only because no data is returned and the check in AnalyticsTestUtils.assertResultGrid doesn't care that no data is present--it only checks correctness for data that is present.

There is no data because it appears that none was ever persisted. The test was never fully written. It parses the test event data but never enters it into the database. See https://github.com/dhis2/dhis2-core/blob/ccee1a68b5075a2b5cfe91c8090b1353025b5995/dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/event/data/EventAnalyticsServiceTest.java#L246-L271 

I spent a bit of time trying to figure out how to persist the test event data, but I couldn't find an easy piece of code to pattern it after. Given enough time and/or guidance I expect I could figure it out, but this is far into a part of the codebase I've never been before. Perhaps it could be done by someone else and/or in a different ticket?